src/core/etl/src/Flow/ETL/DSL/functions.php

```diff
@@ -4,6 +4,55 @@
 
 namespace Flow\ETL\DSL;
 
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Query\QueryBuilder;
+use Doctrine\DBAL\Types\Type as DbalType;
+use Flow\ETL\Adapter\Avro\FlixTech\AvroExtractor;
```

Comment:
IMO adapters should define their own functions definition (it will be handled well on composer autoloader level AFAIK), having them in ETL looks a bit flaky to me.